docs/parameters.md

```diff
@@ -0,0 +1,88 @@
+| `"model_version_policy"` | `json/string` | Optional.The model version policy lets you decide which versions of a model that the OpenVINO Model Server is to serve. By default, the server serves the latest version. One reason to use this argument is to control the server memory consumption.The accepted format is in json or string. Examples: <br> `{"latest": { "num_versions":2 }` <br> `{"specific": { "versions":[1, 3] } }` <br> `{"all": {} }` |
+| `"plugin_config"` | `json/string`  |  List of device plugin parameters. For full list refer to [OpenVINO documentation](https://docs.openvinotoolkit.org/2021.4/openvino_docs_IE_DG_supported_plugins_Supported_Devices.html) and [performance tuning guide](./performance_tuning.md). Example: <br> `{"CPU_THROUGHPUT_STREAMS": "CPU_THROUGHPUT_AUTO"}`  |
+| `"nireq"` | `integer` | The size of internal request queue. When set to 0 or no value is set value is calculated automatically based on available resources.|
+| `"target_device"` | `string` | Device name to be used to execute inference operations. Accepted values are: `"CPU"/"HDDL"/"GPU"/"MYRIAD"/"MULTI"/"HETERO"` |
+| `stateful` | `bool` | If set to true, model is loaded as stateful. | No |
+| `idle_sequence_cleanup` | `bool` | If set to true, model will be subject to periodic sequence cleaner scans.  See [idle sequence cleanup](stateful_models.md#stateful_cleanup). |
+| `max_sequence_number` | `uint32` | Determines how many sequences can be handled concurrently by a model instance. | No |
+| `low_latency_transformation` | `bool` | If set to true, model server will apply [low latency transformation](https://docs.openvinotoolkit.org/2021.4/openvino_docs_IE_DG_network_state_intro.html#lowlatency_transformation) on model load. |
+
+### Batch Processing in OpenVINO Model Server
+
+- `batch_size` parameter is optional. By default, is accepted the batch size derived from the model. It is set by the model optimizer tool.
+- When that parameter is set to numerical value, it is changing the model batch size at service start up. 
+It accepts also a value `auto` - this special phrase make the served model to set the batch size automatically based on the incoming data at run time.
+- Each time the input data change the batch size, the model is reloaded. It might have extra response delay for the first request.
+This feature is useful for sequential inference requests of the same batch size.
+
+*Note:* In case of frequent batch size changes in predict requests, consider using [demultiplexing feature](./demultiplexing.md#dynamic-batch-handling-with-demultiplexing) from [Directed Acyclic Graph Scheduler](./dag_scheduler.md) which is more
```

Comment:
```suggestion
*NOTE*: In case of frequent batch size changes in predict requests, consider using [demultiplexing feature](./demultiplexing.md#dynamic-batch-handling-with-demultiplexing) from [Directed Acyclic Graph Scheduler](./dag_scheduler.md) which is more
```